.travis.yml

```diff
@@ -1,7 +1,7 @@
 # Based on the "trust" template v0.1.2
 # https://github.com/japaric/trust/tree/v0.1.2
 
-dist: trusty
```

Comment:
Ooh, travis has rolled out a newer environment at last.

Reply:
They had to, 14.04 officially lost support almost a month ago :D